support/deployNextFromCI.ts

```diff
@@ -39,11 +39,25 @@ const exec = pify(childProcess.exec);
       console.log(" - prepping package...");
       await exec(`npm run util:prep-next-from-existing-build`);
 
+      // make sure that the changes are committed
+      if ((await exec(`git rev-parse HEAD`)) === (await exec(`git rev-parse origin/master`))) {
```

Comment:
Nitpick: expressive variables can replace the comments and make it easier to update if refactored later